full_utility_rates/California/Ventura CIty of - 28/VENTURA-2016-07-01.owrs

```diff
@@ -0,0 +1,48 @@
+# High level structure for all OWRS templates
+# {} indicates that the value should be substituted from Google sheet
+# [] indicates that the value should be substituted from Google sheet and are mandatory
+# () indicates value to be filled
+---
+metadata:
+  effective_date: 2016-07-01
+  utility_name: "City of Ventura"
+  bill_frequency: bi-monthly
+rate_structure:
+  RESIDENTIAL_SINGLE:
+    tier_starts:
+      - 0
+      - 15
+      - 31
+    tier_prices:
+      - 2.4
+      - 3.35
+      - 5.66
+    commodity_charge: Tiered
+    bill: commodity_charge+service_charge
```

Comment:
bill is commodity_charge + service_charge but no service_charge is provided

Reply:
@vr00n you never provided a service charge :P